src/test/java/org/reaktivity/specification/socks/internal/SocksFunctionsTest.java

```diff
@@ -206,7 +206,7 @@ public void shouldBuildRouteExWithIpv6AddressLeadingZeros() throws Exception
     }
 
     @Test
-    public void shouldBuildRouteExWithIpv6AddressZeroCompression1() throws Exception
+    public void shouldBuildRouteExWithIpv6AddressSimpleZeroCompression() throws Exception
```

Comment:
No need to refer to "zero" compression, since there is only one kind of compression in IPv6, right?

Suggest renaming to `...WithIpv6AddressMiddleCompressedInTwoGroups`.